docs/transactions.md

```diff
@@ -1072,7 +1074,8 @@ and DACs (leaf certificates) added to DCL if they are revoked in the CRL identif
   - pid: `optional(uint16)` -  Product ID (positive non-zero). Must be empty if `IsPAA` is true. Must be equal to a `pid` field in `CRLSignerCertificate`.
   - isPAA: `bool` -  True if the revocation information distribution point relates to a PAA
   - label: `string` -  A label to disambiguate multiple revocation information partitions of a particular issuer.
-  - crlSignerCertificate: `string` - The issuer certificate whose revocation information is provided in the distribution point entry, encoded in X.509v3 PEM format. The corresponding CLI parameter can contain either a PEM string or a path to a file containing the data.
+  - certificate: `string` - The issuer certificate whose revocation information is provided in the distribution point entry, encoded in X.509v3 PEM format. The corresponding CLI parameter can contain either a PEM string or a path to a file containing the data.
```

Comment:
Should the CLI parameter name be changed to `crl-signer-certificate` as well?

Reply:
Not necessary. Let's keep the current name. In general, we don't have a strict rule that CLI parameters have the same name as the corresponding model field (it would be great to have it equal, but it's not due to historical reasons).
If we decide to rename CLI commands and parameters some day, we may consider making all parameters and model fields equal.